package.json

```diff
@@ -38,20 +38,22 @@
     "@fortawesome/free-solid-svg-icons": "^6.1.2",
     "@fortawesome/pro-duotone-svg-icons": "^6.1.2",
     "@fortawesome/pro-solid-svg-icons": "^6.1.2",
-    "@stencil/core": "^2.13.0"
+    "@stencil/core": "^2.13.0",
+    "@types/react": "^18.0.21",
```

Comment:
@alihejazi We still have new type libs for stuff we are not using/ shouldn't be adding